DEVELOPMENT.md

```diff
@@ -54,6 +56,44 @@ _Adding the `upstream` remote sets you up nicely for regularly
 - We've had good success using Docker Desktop: ensure your Kubernetes cluster is healthy and you have plenty of disk space allocated as PVs will be created for PipelineRuns.
 - Ensure you can push images to a Docker registry - the above listed requirements are only for local development, otherwise we pull in the tooling for you in the image.
 
+1. `GOPATH`: If you don't have one, simply pick a directory and add
+   `export GOPATH=...`
+1. `$GOPATH/bin` on `PATH`: This is so that tooling installed via `go get` will
+   work properly.
+1. `KO_DOCKER_REPO`: The docker repository to which developer images should be
```

Comment:
There is also the `ko.local` hack (if you use `minikube` or docker-for-mac) as it doesn't require you to push to a registry, it directly load the images on the k8s docker daemon.
Can be added in a follow-up though :wink:

Reply:
Mb, I only pushed the docs up on my first commit, addressed now

Good point, I remember using `ko.local` and advocated for it when building tektoncd/pipeline, I'm happy for that to be done as a follow-up also (we personally stick to using dockerhub so we needn't spin up and manage our own registry)